### PR TITLE
ign -> gz Header Migration : ign-transport

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/ignition-tooling/release-tools/issues/711

### CODE REVIEW

**Comment by methylDragon** [comment]
CI is pending msgs

I had to install `test_config.h` for `ignition` to run tests properly (without reworking the test CMakeLists)

File: log/include/ignition/transport/log/test_config.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <gz/transport/log/test_config.h>
```
Comment (chapulina) [inline log/include/ignition/transport/log/test_config.h:18]: I think this file isn't installed, so we don't need to tick-tock it

**Review by chapulina — APPROVED** [review]
LGTM once CI is 🟢

File: log/include/ignition/transport/log/test_config.h (hunk 1 above)
Comment (methylDragon) [reply log/include/ignition/transport/log/test_config.h]: https://github.com/ignitionrobotics/ign-transport/pull/310/commits/65128ef46335f47769cfcffb6494a46d45da06f8

I had to change all of the tests (pointing them to `gz`) to allow the package to build without failure with the removal of the ticktocked `ignition/test_config.h`

Comment (methylDragon) [reply log/include/ignition/transport/log/test_config.h]: The relevant exclusion arg has also been removed: https://github.com/ignitionrobotics/ign-transport/pull/310/commits/27bc1869af2f95b93c4a0d16b3b2249909b48459

**Comment by methylDragon** [comment]
Note to self: Pending `ign-msgs`

**Review by chapulina — APPROVED** [review]
👍🏽 with 🟢 CI

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Pull request merged**